README.md

```diff
@@ -64,6 +64,7 @@ $ cd simplefolio
 
 # Remove current origin repository
 $ git remote remove origin
+$ git remote add origin git@github.com:<your github username>/simplefolio.git
```

Comment:
@bfeister Hello! Thank you for making this PR 👏

Is nice to add this, but in these step not.

Because if you add an origin for your username and this moment, later on when you create a repository that will hold your custom Simplefolio project, the repository's name need to be exactly as `simplefolio` because is searching for `simplefolio.git`, so if the repository's name is something like (my-new-website) it will fail when pushing your changes because will not look up the remote repository.

They are more chances to get an error with that. So the thing I am going to do, is to update my video on how deploy Simplefolio and explain steps by steps how to add a custom origin repo.